SwSpotify/__main__.py

```diff
@@ -0,0 +1,14 @@
+from . import spotify, SpotifyNotRunning
```

Comment:
```suggestion
from SwSpotify import spotify, SpotifyNotRunning
```

same thing since local scope is read first but better readability